clmm/modeling.py

```diff
@@ -168,7 +168,7 @@ def predict_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, halo_profi
 def predict_excess_surface_density(r_proj, mdelta, cdelta, cosmo, Delta=200, halo_profile_parameterization='nfw'):
     r"""Computes the excess surface density
 
-    $\Delta\Sigma(R) = \bar{\Sigma}(<R)-\Sigma(R)$, where $\bar{\Sigma}(<R) = \frac{2}{R^2} \int^R_0 dR' R' \Sigma(R')$
+    :math:`\Delta\Sigma(R) = \bar{\Sigma}(<R)-\Sigma(R)`, where :math:`\bar{\Sigma}(<R) = \frac{2}{R^2} \int^R_0 dR' R' \Sigma(R')`
```

Comment:
This shouldn't be inline math.

Inline math ``:math:`equation` ``

Non-Inline math: 
```
..math::
    equation

more text
```
Note that the equation should be tabbed in and there needs to be an after the equation (maybe also before, I don't remember. Compile the docs and take a look)

Reply:
must have an empty line and an extra space before `math`, does not work without it:
```
text

.. math::
    equation

more text
```